### PR TITLE
Fix stickers display text on room summary

### PR DESCRIPTION
**Pull request by surakin**
## Content

Tweak how stickers are displayed in the room summary

## Motivation and context

After this change https://github.com/matrix-org/matrix-rust-sdk/pull/3715 stickers will show up in the room summary but only displaying the content body, which can be interpreted as a file name or as a sticker description, but in both cases it can look like a normal message (Images appear as 'Image').

## Screenshots / GIFs
Before
image
After
image

## Tests

- Step 1
- Step 2
- Step ...

## Tested devices

- [ ] Physical
- [x] Emulator
- OS version(s):

## Checklist

- [x] Changes have been tested on an Android device or Android emulator with API 23
- [ ] UI change has been tested on both light and dark themes
- [ ] Accessibility has been taken into account. See https://github.com/element-hq/element-x-android/blob/develop/CONTRIBUTING.md#accessibility
- [x] Pull request is based on the develop branch
- [x] Pull request title will be used in the release note, it clearly define what will change for the user
- [x] Pull request includes screenshots or videos if containing UI changes
- [x] Pull request includes a [sign off](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#sign-off)
- [x] You've made a self review of your PR

### CODE REVIEW

**Comment by jmartinesp** [comment]
I've created https://github.com/element-hq/element-x-android/pull/3225 to try to fix the CI issues you're seeing. EX Enterprise setup apparently broke it.

**Comment by jmartinesp** [comment]
@surakin that... didn't go well. Maybe you'll need to rebase `origin/develop` 🫤 .

**Comment by jmartinesp** [comment]
Agh, the wrong code got merged... That's what I get for playing with different remotes and having 2 opened PRs...

**Review by jmartinesp — APPROVED** [review]
Now that's better. Let's see if new screenshots are needed, otherwise LGTM.

**Review by jmartinesp — CHANGES_REQUESTED** [review]
It seems like a test case needs to be fixed before it's merged. It would also be nice to check if no screenshot updates are needed.

**Comment by surakin** [comment]
> It seems like a test case needs to be fixed before it's merged. It would also be nice to check if no screenshot updates are needed.

I ran the record screenshot actions and it didn't find any changes :thinking:

File: libraries/eventformatter/impl/src/test/kotlin/io/element/android/libraries/eventformatter/impl/DefaultRoomLastMessageFormatterTest.kt
Comment (jmartinesp) [inline libraries/eventformatter/impl/src/test/kotlin/io/element/android/libraries/eventformatter/impl/DefaultRoomLastMessageFormatterTest.kt:112]: This should be camel cased, the linters will fail otherwise.

**Pull request merged**
